treeherder/perf/migrations/0020_support_backfill_reports.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.6 on 2019-10-25 09:01
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('perf', '0019_remove_confirming_state'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BackfillReport',
```

Comment:
Do we expire old alert summaries/alerts and will these tables get cleaned up periodically by the cycle_data or other task?

Reply:
We're not expiring alert summaries/alerts, because there are still Bugzilla comments pointing back to them via hyperlinks.
Regarding the cleanup of these reports: we will consider this in our future roadmaps, once we turn on the auto retriggering bot. Thing is the amount of reports & records won't ever exceed that of alerts summaries & alerts.

Reply:
> We're not expiring alert summaries/alerts, because there are still Bugzilla comments pointing back to them via hyperlinks.
> 
> Regarding the cleanup of these reports: we will consider this in our future roadmaps, once we turn on the auto retriggering bot. We need to know to which extent we can expire them, without affecting the bot.

Yes, please do add that to your roadmap. We already store a lot of historical data for Perfherder and I'd like to be sure as we add new tables or new columns to tables we have a plan for the data that is no longer useful.

Reply:
@davehunt we need to consider expiring the alert summaries & alerts also.

Reply:
Could you check with BMO to see if there is a policy regarding how long linked resources should remain available? I think it's reasonable to show a page stating that the alert is no longer available, but not sure what age we should do this at.